Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -346,7 +345,6 @@ - (void)handleLocalNotificationReceived:(NSNotification *)notification
                                                  categories:categories];
 
     [RCTSharedApplication() registerUserNotificationSettings:settings];
-    [RCTSharedApplication() registerForRemoteNotifications];
```

Comment:
Yes, `registerUserNotificationSettings` has a callback that calls `registerForRemoteNotifications`. https://github.com/iodine/react-native/blob/marc.fix-notifs/Libraries/PushNotificationIOS/RCTPushNotificationManager.m#L101

The double registration seemed to be the reason for the notifs showing twice